.gitignore

```diff
@@ -11,4 +11,4 @@ __pycache__
 # Editor files
 *.swp
 .idea
-*.iml
+*.iml
```

Comment:
Can you rever this? It is a minor thing but everything that is not a recipe should not be modified in staged-recipes.